__tests__/integration.js

```diff
@@ -59,6 +59,11 @@ const PORT_RANGE = MAX_PORT_NUM - MIN_PORT_NUM;
 const getRandomPort = () => Math.floor(Math.random() * PORT_RANGE) + MIN_PORT_NUM;
 
 async function runYarn(args: Array<string> = [], options: Object = {}): Promise<Array<Buffer>> {
+  if (!options['env']) {
+    options['env'] = Object.assign({}, process.env);
+    options['extendEnv'] = false;
```

Comment:
What is this? I've searched the code and couldn't find any references to `extendEnv` :(

Reply:
`extendEnv` is an option of `execa` function. Without passing this option set to false `execa` executes the following piece of code:

```javascript
if (opts && opts.env && opts.extendEnv !== false) {
	opts.env = Object.assign({}, process.env, opts.env);
}
```

This merges `process.env` and `opts.env` and FORCE_COLOR is set again.